projects/vdk-core/src/vdk/internal/builtin_plugins/connection/managed_cursor.py

```diff
@@ -156,6 +152,15 @@ def _decorate_operation(self, managed_operation: ManagedOperation, operation: st
                 self.__connection_hook_spec.db_connection_decorate_operation
             )
 
+        # apply any changes to the managed operation from non-db plugins
```

Comment:
But db-plugins would be able to use those hooks, too? Am I wrong?

Reply:
You're correct, I'll change the comment.